js/privatebin.js

```diff
@@ -2762,9 +2762,13 @@ jQuery.PrivateBin = (function($) {
 
             if (format === 'markdown') {
                 $plainText.removeClass('hidden');
+                // Show copy button for markdown format
+                $('#prettyMessageCopyBtn').removeClass('hidden');
```

Comment:
Since we are doing the same in both cases, I would move this line to just after the if-else-block (or before it). This to avoid duplicate code.